File: Java Word Examples/MailMergeApiDemo.java
import spire.cloud.word.sdk.client.*;
import spire.cloud.word.sdk.client.api.MailMergeApi;
import spire.cloud.word.sdk.client.model.MailMergeData;
import java.util.*;

public class MailMergeApiDemo {
    static String appId = "your id";
    static String appKey = "your key";
    static String baseUrl = "https://api.e-iceblue.cn";
    static Configuration wordConfiguration = new Configuration(appId, appKey, baseUrl);
    static MailMergeApi mailMergeApi = new MailMergeApi(wordConfiguration);

    public static void mailMerge() throws ApiException {
        String name = "mailMerge.doc";
        List<MailMergeData> data = new ArrayList<MailMergeData>();
        data.add(new MailMergeData("Contact Name", "E-iceblue"));
        data.add(new MailMergeData("Fax", "TestFax"));
        data.add(new MailMergeData("Date", "2020-5-10"));
        String folder = "input";
        String storage = null;
        String password = null;
        String destFileName = "output/mailMerge_output.docx";
        mailMergeApi.mailMerge(name, destFileName, data, folder, storage, password);
    }

}

File: Java Word Examples/MailMergeApiDemo.cs
using Spire.Cloud.Word.Sdk.Api;
using Spire.Cloud.Word.Sdk.Client;
using Spire.Cloud.Word.Sdk.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CloudWord
{
    class MailMergeApiDemo
    {
        static string appId = "your id";
        static string appKey = "your key";
        static string baseUrl = "https://api.e-iceblue.cn";
        static Configuration wordConfiguration = new Configuration(appId, appKey, baseUrl);
        static MailMergeApi mailMergeApi = new MailMergeApi(wordConfiguration);
        public static void MailMerge()
        {
            string name = "mailMerge.doc";
            List<MailMergeData> data = new List<MailMergeData>();
            data.Add(new MailMergeData("Contact Name", "E-iceblue"));
            data.Add(new MailMergeData("Fax", "TestFax"));
            data.Add(new MailMergeData("Date", "2020-5-10"));
            string folder = "input";
            string storage = null;
            string password = null;
            string destFileName = "output/mailMerge_output.docx";
            mailMergeApi.MailMerge(name, destFileName, data, folder, storage, password);
        }

    }
}
